### DISCUSSION TITLE
Renaming the master branch to a more inclusive term

### DISCUSSION THREAD

**Discussion by luminitavoicu** [firecracker-microvm/firecracker#2509] [General]
Firecracker strives to provide an environment that is inclusive and welcoming for all people. 

We have previously updated our code and documentation to use more inclusive terms. We want to continue this process by changing the name of our master branch to _main_, in about two weeks' time.

We are eager to receive your feedback! We also want to make sure this change can be consumed in all existing code workflows - let us know otherwise.

**Comment by alsrdn** [comment]
This is already implemented. Master branch was renamed to main.
